data/pizza.json

```diff
@@ -1130,7 +1118,7 @@
 			"ingredients": [
 				"isbergssallad", "tomat", "gurka", "feferoni", "lök", "kebabkött", "pommes frites eller ris"
```

Comment:
[Should include kebabsås](https://sannegarden.com/meny/johanneberg/#:~:text=L%C3%B6k%2C%20Kebabk%C3%B6tt%20(Fl%C3%A4skkarr%C3%A9)%2C%20Feferoni%2C%20Kebabs%C3%A5s%20Mild%2C%20Gurka%2C%20Isbergssallad%2C%20Pommes%20Frites%2C%20Tomater)